ci-operator/step-registry/ipi/conf/aws/blackholenetwork/ipi-conf-aws-blackholenetwork-commands.sh

```diff
@@ -303,19 +305,18 @@ EOF
 
 MAX_ZONES_COUNT="$(cat "${SHARED_DIR}/maxzonescount")"
 
-ZONE_COUNT=3
+ZONES_COUNT=${ZONES_COUNT:-2}
 if [[ "${MAX_ZONES_COUNT}" -lt 3 ]]
-	
 then
-  ZONE_COUNT="${MAX_ZONES_COUNT}"
+  ZONES_COUNT="${MAX_ZONES_COUNT}"
 fi
 
 STACK_NAME="${CLUSTER_NAME}-shared-vpc-blackhole"
 aws --region "${REGION}" cloudformation create-stack \
   --stack-name "${STACK_NAME}" \
   --template-body "$(cat /tmp/blackhole_vpc.yaml)" \
   --tags "${TAGS}" \
-  --parameters "ParameterKey=AvailabilityZoneCount,ParameterValue=${ZONE_COUNT}" &
+  --parameters "ParameterKey=AvailabilityZoneCount,ParameterValue=${ZONES_COUNT}" &
```

Comment:
nit: since we're using this to fill `AvailabilityZoneCount`, I don't think we need to change `ZONE_COUNT` to `ZONES_COUNT`.